XDR iocs bug (#13763)
demisto/content: Packs/CortexXDR/ReleaseNotes/3_0_20.md

#### Integrations
##### Cortex XDR - IOC
- Updated the Docker image to: *demisto/python3:3.9.6.22912*.

> - Fixed an issue with ***xdr-iocs-sync*** (also relevant for ***fetch-incidents*** if you are using `Auto sync`), which caused authentication erros.

Reviewer: ```suggestion
- Fixed an issue with the ***xdr-iocs-sync*** and ***fetch-incidents*** commands where customers using `Auto sync` would encounter authentication errors.
```
